tests/TestDocs.hs

```diff
@@ -29,13 +30,15 @@ publishOpts = Publish.defaultPublishOptions
   where testVersion = ("v999.0.0", Version [999,0,0] [])
 
 main :: IO ()
-main = do
-  pushd "examples/docs" $ do
-    Docs.Package{..} <- Publish.unsafePreparePackage publishOpts
-    forM_ testCases $ \(P.moduleNameFromString -> mn, pragmas) ->
-      let mdl = takeJust ("module not found in docs: " ++ P.runModuleName mn)
-                         (find ((==) mn . Docs.modName) pkgModules)
-      in forM_ pragmas (flip runAssertionIO mdl)
+main = pushd "examples/docs" $ do
+  res <- Publish.preparePackage publishOpts
+  case res of
+    Left e -> preparePackageError e
```

Comment:
Oh, of course, you're right. Sorry. I forgot that most of the tests work by just calling `exitFailure` rather than using some testing framework. I think we want to do the same as the unsafe version except print to standard output instead of standard error (because the rest of the tests print to stdout). You might have seen a travis build where the actual error is muddled up with some of the compiler test suite output because they're printing to different streams.